src/d3d11/d3d11_annotation.h

```diff
@@ -33,6 +33,9 @@ namespace dxvk {
 
     ID3D11DeviceContext*  m_container;
 
+    // Stack depth for non-finalized BeginEvent calls
+    int m_eventDepth;
```

Comment:
is there a reason why this is signed? (also, prefer `int32_t` or `uint32_t`)

Reply:
I guess because D3D returns this signed

Reply:
Sure. I'll make this a sized type. Josh is correct, it's signed because that's how the D3D interface wants it. Basically it's a stack-like behavior of events being pushed/popped and if we have a negative number that indicates an error from the application ending more event regions than it began.